src/linear_algebra/multilinear/basic.lean

```diff
@@ -676,6 +676,43 @@ def dom_dom_congr_linear_equiv {ι₁ ι₂} [decidable_eq ι₁] [decidable_eq
   .. (dom_dom_congr_equiv σ : multilinear_map A (λ i : ι₁, M₂) M₃ ≃+
         multilinear_map A (λ i : ι₂, M₂) M₃) }
 
+/-- The dependent version of `multilinear_map.dom_dom_congr_linear_equiv`. -/
+@[simps apply symm_apply]
+def dom_dom_congr_linear_equiv' {ι' : Type*} [decidable_eq ι'] (σ : ι ≃ ι') :
+  multilinear_map R M₁ M₂ ≃ₗ[R] multilinear_map R (λ i, M₁ (σ.symm i)) M₂ :=
+{ to_fun    := λ f,
+    { to_fun    := f ∘ (σ.Pi_congr_left' M₁).symm,
+      map_add'  := λ m i,
+        begin
```

Comment:
Here I would also keep the indentation of `begin` flush with that of the `map_add'` on the line above it. So a total of two indentation levels.